vignettes/estimate_from_individual_data.Rmd

```diff
@@ -0,0 +1,321 @@
+  )
+
+# Create plot with lines linking onset and outcome
+ggplot() +
+  geom_segment(
+    data = data, aes(x = case_onsets,
+                     xend = outcome_dates,
+                     y = id_ordered,
+                     yend = id_ordered),
+    color = ifelse(data$outcome_type == "D", "red", "green"),
+    size = 0.5
+  ) +
+  geom_point(data = plot_data, aes(x = date,
+                                   y = id_ordered,
+                                   color = outcome_type,
+                                   shape = event_label),
+             size = 2) +
+  geom_vline(xintercept = as.numeric(max_obs),
+             linetype = "dashed",
+             color = "black",
+             size = 0.5) +
+  scale_color_manual(values = c(D = "darkred", R = "darkgreen")) +
+  scale_shape_manual(values = c(Onset = 16, Outcome = 17)) +
+  labs(
+    x = "Date",
+    y = "Individual (Ordered by onset date)",
+    color = "Outcome type",
+    shape = "Event type"
+  ) +
+  theme_minimal()
+```
+
+First, we calculate CFR by filtering to focus only on cases with known outcomes:
+```{r}
+# Filter on known outcomes
+total_deaths <- sum(known_outcomes == "D")
+total_outcomes <- (total_deaths + sum(known_outcomes == "R"))
+
+# Calculate CFR with 95% CI
+cfr_filter <- binom.test(total_deaths, total_outcomes)
+cfr_estimate <- (signif(as.numeric(c(cfr_filter$estimate,
+                                     cfr_filter$conf.int)), 3))
+cfr_estimate
+```
+
+Next, we calculate CFR based on expected fatal outcomes known to date, using `cfr_static()`
+```{r}
+# Get times of death for fatal outcomes
+death_times <- (case_onsets)[type_outcome == "D"] + outcome_death
+
+# Create a single data.frame with event types
+events <- data.frame(
+  dates = c(case_onsets, death_times),
+  event = c(rep("cases", length(case_onsets)),
+            rep("deaths", length(death_times)))
+)
+
+# Use incidence2 to calculate counts of cases and deaths by day
+counts <- incidence2::incidence(events,
+                                date_index = "dates",
+                                groups = "event",
+                                complete_dates = TRUE)
+
+# Pivot incidence object to get data.frame with counts for cases and deaths
+df <- counts %>%
+  tidyr::pivot_wider(names_from = event,
+                     values_from = count,
+                     values_fill = 0) %>%
+  dplyr::rename(date = date_index)
+
+cfr_static(df, delay_death)
+```
+Hence in this particular simulation, the `cfr_static()` method recovers the correct CFR of 10% (95% CI: 8.3-12.0%), whereas the filtering method produces a biased estimate of 26.4% (17.6-37.0%).
+
+## Deaths reported but not recoveries
+
+In an extreme scenario where recoveries are not reported, then we effectively have the values of `outcome_recovery` generated from a distribution with an infinite mean, and the above methods will still apply, with the same bias for the filtering approach. In particular, we would expect:
```

Comment:
but doesn't this mean that the filtering method is not really recommended in this scenario at all, and that users should use cfr_static as id doesn't rely on reported recoveries?

Reply:
Good point - have swapped 'methods' to 'conclusions'